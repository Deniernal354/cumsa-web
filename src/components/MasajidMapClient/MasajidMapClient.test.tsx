import { render } from "@testing-library/react";
import MasajidMapClient from "./MasajidMapClient";

test("MasajidMapClient snapshot", () => {
  const { asFragment } = render(<MasajidMapClient mosques={[]} />);
  expect(asFragment()).toMatchSnapshot();
});
