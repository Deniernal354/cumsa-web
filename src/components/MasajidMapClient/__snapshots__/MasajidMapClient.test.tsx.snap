// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MasajidMapClient snapshot 1`] = `
<DocumentFragment>
  <div
    class="p-4 text-sm text-gray-600"
  >
    No masajid available.
  </div>
</DocumentFragment>
`;
